doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3072,10 +3075,10 @@ Conversion tactics applied to hypotheses
    listed in this section.
 
    If :n:`@ident` is a local definition, then :n:`@ident` can be replaced by
-   (Type of :n:`@ident`) to address not the body but the type of the local
+   (type of :n:`@ident`) to address not the body but the type of the local
```

Comment:
I had missed this: it would be even better to put the verbatim quotes around the whole expression:
```
:n:`(type of @ident)`
```